tools/ci_build/github/azure-pipelines/templates/android-java-api-aar.yml

```diff
@@ -40,69 +40,71 @@ parameters:
   displayName: Package Name
   type: string
   default: 'onnxruntime-android'
-
-jobs:
-- job: Android_Java_API_AAR_Packaging_${{ parameters.job_name_suffix }}
-  timeoutInMinutes: 120
-  workspace:
-    clean: all
-  pool: ${{parameters.pool_name}}
-
-  variables:
-    artifacts_directory: $(Build.BinariesDirectory)/.artifacts
-
-  steps:
-  - checkout: self
-    clean: true
-    submodules: none
-
-  - task: CmdLine@2
-    displayName: Create artifacts directory
-    inputs:
-      script: |
-        # Create a folder for artifacts
-        mkdir -p $(artifacts_directory)
-      workingDirectory: $(Build.BinariesDirectory)
-
-  - template: get-docker-image-steps.yml
-    parameters:
-      Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cpu
-      Context: tools/ci_build/github/linux/docker
-      DockerBuildArgs: "--build-arg BUILD_UID=$( id -u )"
-      Repository: onnxruntimecpubuild
-
-  - template: set-version-number-variables-step.yml
-
-  - template: use-android-ndk.yml
-
-  - task: CmdLine@2
-    displayName: Build Android AAR Packages
-    inputs:
-      script: |
-        set -e -x
-        NDK_HOME=$(realpath $ANDROID_NDK_HOME)
-        mkdir $(Build.BinariesDirectory)/.build_settings
-        cp ${{parameters.buildSettings}} $(Build.BinariesDirectory)/.build_settings/build_settings.json
-        [ -f "${{parameters.includedOpsConfig}}" ] && \
-          cp ${{parameters.includedOpsConfig}} $(Build.BinariesDirectory)/.build_settings/include_ops_and_types.config
-        docker run --rm \
-          --volume $(Build.SourcesDirectory):/onnxruntime_src \
-          --volume $(Build.BinariesDirectory):/build \
-          --volume $ANDROID_HOME:/android_home \
-          --volume $NDK_HOME:/ndk_home \
-          --volume $(artifacts_directory):/home/onnxruntimedev/.artifacts \
-          --volume $(Build.BinariesDirectory)/.build_settings:/home/onnxruntimedev/.build_settings \
-          -e NIGHTLY_BUILD \
-          -e BUILD_BUILDNUMBER \
-          -e BUILD_CONFIG=${{parameters.buildConfig}} \
-          -e ORT_VERSION=$(OnnxRuntimeVersion) \
-          -e PUBLISH_EXECUTABLES=${{parameters.publish_executables}} \
-          -e PACKAGE_NAME=${{parameters.packageName}} \
-          onnxruntimecpubuild \
-            /bin/bash /onnxruntime_src/tools/ci_build/github/android/build_aar_and_copy_artifacts.sh
-      workingDirectory: $(Build.SourcesDirectory)
-
-  - task: PublishBuildArtifacts@1
-    inputs:
-      pathtoPublish: '$(artifacts_directory)'
-      artifactName: '${{parameters.artifactName}}'
+stages:
```

Comment:
will need to update other usages of this template too, e.g.:
https://github.com/microsoft/onnxruntime/blob/bd142bfb04db00253265203fb0af2988e751d9a0/tools/ci_build/github/azure-pipelines/build-perf-test-binaries-pipeline.yml#L24
https://github.com/microsoft/onnxruntime/blob/bd142bfb04db00253265203fb0af2988e751d9a0/tools/ci_build/github/azure-pipelines/templates/react-native-ci.yml#L24